doc/Type/IO/Path/Win32.pod6

```diff
@@ -27,8 +27,9 @@ Defined as:
 Returns a string that, when given passed through L«C<EVAL>|/routine/EVAL»
 gives the original invocant back.
 
-    IO::Path::Win32.new("foo/bar").perl.say;
-    # OUTPUT: IO::Path::Win32.new("foo/bar", :CWD("C:\\Users\\camelia"))
+=for code :ok-test<perl>
```

Comment:
Sorry, what is this `:ok-test<perl>`? If it's OK to test, you don't need to write anything.